arm-cdn/2016-10-02/swagger/cdn.json

```diff
@@ -1750,11 +1750,19 @@
         },
         "httpPort": {
           "description": "The value of the HTTP port. Must be between 1 and 65535",
-          "type": "integer"
+          "type": "integer",
+		  "maximum": 65535,
+		  "exclusiveMaximum": false,
```

Comment:
false value for exclusiveMaximum and exclusiveMinimum is redundant.

The spec says:
> if "exclusiveMaximum" is **not present, or has boolean value false**, then the instance is valid if it is lower than, or equal to, the value of "maximum";